### PR TITLE
length is not always available

### PR DESCRIPTION
**Pull request by takethefake**

### CODE REVIEW

File: src/Admin/Model/DoctrineAutodiscoveryModel.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function fetchFields($module, $version, $adapter_name)
                 switch ($mapping['type']) {
                     case 'string':
                         $field['filters'] = $this->filters['text'];
-                        if ($mapping['length'] != '') {
+                        if (isset($mapping['length'])) {
```
Comment (TomHAnderson) [inline src/Admin/Model/DoctrineAutodiscoveryModel.php]: Please change to 
```
if (isset($mapping['length']) && $mapping['length']) {
```

**Comment by jguittard** [comment]
Please fix the the CS issues and resubmit your PR please

**Comment by takethefake** [comment]
feels like i'm to Dumb for this

**Comment by TomHAnderson** [comment]
@takethefake you're just faking.  The CS issues are, infact, not something directly associated with your PR but as the changes are new (phpcs upgrade?) please do us a favor and get the PR passed (thereby giving you more code in the app than you'd otherwise be submitting).

**Comment by takethefake** [comment]
I'm really really sorry. but this collaboration work is new for me. I Hope that it is finally done. And there is no need to argue.

**Pull request merged**